public/data/subhadiptech.json

```
@@ -0,0 +1,17 @@
+{
+    "name": "Subhadip Das",
+    "bio": "Improving",
+    "avatar": "https://avatars.githubusercontent.com/u/89024718?s=400&u=ff742340a81589cb9dd85fe7c978a68c9e17ae56&v=4",
```

HI @Subhadiptech, Welcome to the community.
Can you please update your avatar link to:
```suggestion
    "avatar": "https://github.com/Subhadiptech.png",
```
so that whenever you will update your profile It will automatically capture the latest one.

---

Thanks for the suggestions @all